napari_tiff/napari_tiff_metadata.py

```diff
@@ -243,7 +238,7 @@ def get_imagej_metadata(tif: TiffFile) -> dict[str, Any]:
             rgb = False
             n = shape[channel_axis]
             visible = [True, True, True]
-            colormap = rgb_colormaps(samples=4)[:n]
+            colormap = [ALL_COLORMAPS["red"], ALL_COLORMAPS["green"], ALL_COLORMAPS["blue"]]
             name = ["Red", "Green", "Blue", "Alpha"][:n]
             blending = ["additive", "additive", "additive", "translucent"][:n]
```

Comment:
why transcluent?

Reply:
I'm not sure of the original logic behind this particular decision, but I'm going to guess it has something to do with the fact the last channel is the alpha transparency channel. Maybe best to grab an example image and look at the difference in behaviour.